modules/templates/SSF/config.py

```diff
@@ -539,6 +539,34 @@ def task_rheader(r):
             return rheader
 
         attr["rheader"] = task_rheader
+
+        standard_postp = s3.postp
+
+        def postp(r, output):
+            if callable(standard_postp):
+                output = standard_postp(r, output)
+
+            if auth.is_logged_in():
+                filter_url = URL(c="project", f="task", args=[])
+                filter_url += "?~.project_member.person_id__eq=%s"
```

Comment:
URL takes a dict for vars, like vars={"a":"b"} then URL = something/something?a=b

But the URL Query isn't a get variable...